features/step_definitions/content_steps.rb

```diff
@@ -23,6 +23,17 @@
     fail unless page.has_content?(arg1)
   end
 end
+
+#
+# Test for a text in the whole page
+#
+Then(/^I should see a "([^"]*)" or "([^"]*)" text$/) do |text, text2|
+  unless page.has_content?(text) || page.has_content?(text2)
+    sleep 25
```

Comment:
@meaksh  stuff there is when maybe we need to wait for a javascript code to be executed, we can make it to 10 sec imho